### PR TITLE
Create: Add helper text for Project details

### PR DESCRIPTION
**Pull request by wraeth-eth**
## What does this PR do and why?

_Provide a description of what this PR does. Link to any relevant GitHub issues, Notion tasks or Discord discussions._

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by wraeth-eth** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2510** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2510" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2510?utm_source=stack-comment).

File: src/components/Create/components/pages/ProjectDetails/ProjectDetailsPage.tsx
Comment (tomquirk) [inline src/components/Create/components/pages/ProjectDetails/ProjectDetailsPage.tsx:111]:
```suggestion
                    tip={t`Display a message or warning to contributors before they pay your project`}
```

**Review by tomquirk — APPROVED** [review]

**Comment by wraeth-eth** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2510?utm_source=stack-comment).


Job Reference: XJjT605R1Sdi2CUgZUbX

**Pull request merged**
